### PR TITLE
Imprint Creation

### PR DESCRIPTION
**Pull request by AdrianBor**
I’ve a quick issue with the link from ProfilDetail.js… I can’t make the link… I really don’t know why…. Could you fix it ?

### CODE REVIEW

File: src/pages/ProfilDetail.js [hunk 1]
```diff
@@ -68,6 +71,7 @@ const ProfilDetailContainer = connect(
   (state, ownProps) => ({}),
   (dispatch) => ({
     showTrip: bindActionCreators(goToTrip, dispatch),
+    ShowImprint: bindActionCreators(goToImprint, dispatch),
```
Comment (lukaswelte) [inline src/pages/ProfilDetail.js]: this s of `ShowImprint` needs to be lower case so `showImprint`

File: src/pages/ImprintDetail.css [hunk 2]
```diff
@@ -0,0 +1,222 @@
+.ImprintDetail-card{
```
Comment (lukaswelte) [inline src/pages/ImprintDetail.css:1]: You need to clean the css to just have the classes needed for the page

**Review by lukaswelte — CHANGES_REQUESTED** [review]

File: src/pages/ImprintDetail.css (hunk 2 above)
Comment (AdrianBor) [reply src/pages/ImprintDetail.css:1]: ok!

File: src/pages/ProfilDetail.js (hunk 1 above)
Comment (AdrianBor) [reply src/pages/ProfilDetail.js]: ok :)

**Comment by AdrianBor** [comment]
@lukaswelte Changements are done.

**Comment by lukaswelte** [comment]
That is great. Can we have this button also on login and introduction @AdrianBor?

**Review by lukaswelte — APPROVED** [review]

**Comment by AdrianBor** [comment]
ok @lukaswelte I'm done, I just have an issue with the styling on the introduction page. I'm trying to do that but doesn't work...

LegalButton onClick={showImprint} style={{color:'#252525'}}/

could you tell me what's missing ?

**Comment by lukaswelte** [comment]
To be able to tell you what is missing i need to know what your problem is ;) 

Maybe just that the text is white on a white background?

**Comment by AdrianBor** [comment]
Totally right. The text is white on white and for this specific case i try to change the color to black directly on the page.

File: src/components/LegalButton.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React, { PureComponent } from 'react';
+import './LegalButton.css';
+
+export default class LegalButton extends PureComponent {
+  render() {
+    const { onClick } = this.props;
```
Comment (lukaswelte) [inline src/components/LegalButton.js]: pass a property color in here.
Then apply an inline style that sets the color. Thereby you can pass the desired color into the component

**Comment by AdrianBor** [comment]
Ok @lukaswelte I've to go right now, I tried to do it but I've still the problem. I don't know where to Call the color style. I guess I'm very close but...

Sorry for behing so bad! Soon no more code for us maybe! 💃 💃 💃

**Comment by AdrianBor** [comment]
ok @lukaswelte , done! thanks a lot.

**Pull request merged**